src/components/SnapshotScheduleInput.tsx

```diff
@@ -3,27 +3,27 @@ import { Input, RadioInput, Select } from "@canonical/react-components";
 
 const snapshotOptions = [
   {
-    label: "minute",
+    label: "Every minute",
```

Comment:
This looks a bit odd, having one option starting with uppercase and the others all lowercase. Shall we also write "Hourly", "Daily", etc?

Reply:
This change resolves the issue of multiple labels pointing at the same form control since we have the configuration title that points at the `Input` or `Select` components but at the same time these components have their own labels. I'm sure you are already aware of this, just leaving the above comment here for future reference if we ever look back at the PR and try to figure out why this was done :slightly_smiling_face: 

I agree with the above comment, it looks a bit weird only one option starts with uppercase, will change the other options to look the same.